tools/dds/dds-server/dds-device-broadcaster.cpp

```diff
@@ -175,13 +157,24 @@ bool dds_server::add_dds_device( const std::string & device_key,
     return true;
 }
 
-bool dds_server::create_device_writer( const std::string &device_key, rs2::device rs2_device )
+bool dds_device_broadcaster::create_device_writer( const std::string &device_key, rs2::device rs2_device )
 {
     // Create a data writer for the topic
     DataWriterQos wqos = DATAWRITER_QOS_DEFAULT;
     wqos.reliability().kind = RELIABLE_RELIABILITY_QOS;
     wqos.durability().kind = VOLATILE_DURABILITY_QOS;
-    wqos.data_sharing().automatic();
+
+    //---------------------------------------------------------------------------------------
```

Comment:
@maloel Please review this new addition which replaced the `sleep` we has before the `writer.write()`